api/definition/fingertipsAPI.yaml

```
@@ -879,6 +880,23 @@ components:
               description: The value used in the comparison
         deprivation:
           $ref: '#/components/schemas/DeprivationData'
+    IndicatorSegment:
+      title: IndicatorSegment
+      type: object
+      description: Represents a segment of an Indicator health data point for a public health indicator .
+      required:
+        - sex
+      properties:
+        sex:
+          $ref: '#/components/schemas/SexData'
```

as mentioned previously could this just be 
```suggestion
          $ref: '#/components/schemas/SegmentInfo'
```